linkerd/meshtls/boring/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "linkerd-meshtls-boring"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+license = "Apache-2.0"
+edition = "2018"
+publish = false
+
+[dependencies]
+boring = { version = "1" }
+boring-sys = { version = "1"}
+futures = { version = "0.3", default-features = false }
+linkerd-error = { path = "../../error" }
+linkerd-dns-name = { path = "../../dns/name" }
+linkerd-identity = { path = "../../identity" }
+linkerd-io = { path = "../../io" }
+linkerd-stack = { path = "../../stack" }
+linkerd-tls = { path = "../../tls" }
+thiserror = "1"
```

Comment:
looks like we never actually use the `thiserror` dependency in this crate --- are we planning to?